### PR TITLE
Add clarification about date component validation

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) adds clarification about the required parameter for date validation.

*a similar clarification was added to a separate topic within this PR - https://github.com/magento/devdocs/pull/8874

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/ui_comp_guide/components/ui-date.html

## Links to Magento source code

-https://github.com/magento/magento2/blob/2.4/lib/web/mage/validation.js#L1046

### CODE REVIEW

**Comment by EvheniiaKolkhiieva** [comment]
Re: This notice will be added to the `2.4 `version of this document with the next commit.

File: src/guides/v2.3/ui_comp_guide/components/ui-date.md [hunk 1]
```diff
@@ -55,6 +55,28 @@ Extends [`Abstract`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/M
 </form>
 ```
 
+## Validation
+
+There is the **validate-date** initialization rule, that describes the required **dateFormat** parameter, should be specified:
+
+```javascript
+'validate-date': [
```
Comment (BarnyShergold) [inline src/guides/v2.3/ui_comp_guide/components/ui-date.md:63]: This JS snippet is not indented properly as I see it.

File: src/guides/v2.3/ui_comp_guide/components/ui-date.md [hunk 2]
```diff
@@ -55,6 +55,28 @@ Extends [`Abstract`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/M
...
+                var test = moment(value, utils.convertToMomentFormat(additionalParams.dateFormat));
+
+                return $.mage.isEmptyNoTrim(value) || test.isValid();
+            },
+            $.mage.__('Please enter a valid date.')
+
+        ];
+```
+
+The following data shows how to add date validation using a calendar widget as an example.
```
Comment (BarnyShergold) [inline src/guides/v2.3/ui_comp_guide/components/ui-date.md:74]:
```suggestion
The following shows how to add date validation using a calendar widget as an example.
```

File: src/guides/v2.3/ui_comp_guide/components/ui-date.md [hunk 3]
```diff
@@ -55,6 +55,28 @@ Extends [`Abstract`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/M
 </form>
 ```
 
+## Validation
+
+There is the **validate-date** initialization rule, that describes the required **dateFormat** parameter, should be specified:
```
Comment (BarnyShergold) [inline src/guides/v2.3/ui_comp_guide/components/ui-date.md:60]:
```suggestion
The **validate-date** initialization rule, that describes the required **dateFormat** parameter, should be specified:
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a few corrections suggested

File: src/guides/v2.3/ui_comp_guide/components/ui-date.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/guides/v2.3/ui_comp_guide/components/ui-date.md]: Hi @BarnyShergold, thank you for your review! 

I have implemented your suggestions, could you please review and approve them?

Thank you! 😊

File: src/guides/v2.3/ui_comp_guide/components/ui-date.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/guides/v2.3/ui_comp_guide/components/ui-date.md]: This is done, please check!

File: src/guides/v2.3/ui_comp_guide/components/ui-date.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/guides/v2.3/ui_comp_guide/components/ui-date.md]: This is done, please check!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8898`.

**Pull request merged**